node/src/GlideClusterClient.ts

```diff
@@ -971,16 +1029,32 @@ export class GlideClusterClient extends BaseClient {
     public async functionList(
         options?: FunctionListOptions & DecoderOption & RouteOption,
     ): Promise<ClusterResponse<FunctionListResponse>> {
-        return this.createWritePromise(createFunctionList(options), {
+        return this.createWritePromise<
+            GlideRecord<unknown> | GlideRecord<unknown>[]
+        >(createFunctionList(options), {
             route: toProtobufRoute(options?.route),
             decoder: options?.decoder,
-        });
+        }).then((res) =>
+            res.length == 0
```

Comment:
```suggestion
            res
```

Reply:
That will be null/undef check, but I need empty check